### PR TITLE
FISH-7044: preventing the addition of multiple HSTS header for response

### PR DESCRIPTION
**Pull request by breakponchito**
Adding fix to prevent multiple security headers of Strict-Transport-Security on the response
## Description

This is a fix for the following reported bug: https://github.com/payara/Payara/issues/6178
### Testing Performed

Manual testing with the reproducer included on the ticket:  https://payara.atlassian.net/browse/FISH-7044
Now the request is working fine adding just one occurrence of the security header on the response:
`
8m2pcqb77qu6lh5pl7v35pgyhauot2ntazall9f4idjs5lypnu5u83k6tf165ijyl64r0nf8mm1kexkclstpr13xgtmidw25pjv6yx6w2qyvm6jxu2vduxtj1mxsfralqbs4hss1ka6cpxdejq1nxly97p81zcqhpg22952l4rt6pk30qkm6f96ob7h33iines80f2bj6ziwwlimewj2az7rsoiae1nwskyyde4uh9l1xebng8e0vyr68pg3kc8l8067cjqy6lblvuk2xc5oiw01lasqnzi9vmrokal7scnik13x8xyuv1r54btm7mdxt97pu0xydhj2ahrrsnqgqx1ydsm41wyyh7de01zy0ktufu6okdouzkk2hnz94ywbjywsb7smwz2dffirm4nn36ah3nggvftwt8is6xl7xk8mj2phe0gl8gro4zz0xlrafub3aqom7xoen29rkevcsdgtm6cyegityc6qnoxqowoum69bnhwi1d9xjn5p9l6ax683kjah7988nlr4t1mhrw9sgce3vbn571xry92w1hfdlqct5atwfmyplkxvnx5s4lv5mww910mkj8ojykm634wm2jnfqrfvku6vpuky0oxchxpjl3napjy40hdkfk24nz52dr6vumh4ys7eu05ozbd3rv15dvld3glmkox0w8< HTTP/2 200

< strict-transport-security: max-age=31536000
< content-type: text/plain
`
### Testing Environment

Windows 11, Azul jdk 11, maven 3.8.3

### CODE REVIEW

**Comment by breakponchito** [comment]
Jenkins test please

**Comment by breakponchito** [comment]
Jenkins test please

**Comment by breakponchito** [comment]
Jenkins test please

**Review by Pandrex247 — APPROVED** [review]

File: nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/HSTSFilter.java
Comment (Pandrex247) [inline nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/HSTSFilter.java:1]: Copyright needs updating

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**